### PR TITLE
HUB-30: Implement changes for the mobile segments AB test

### PR DESCRIPTION
**Pull request by CharlesIC**
Author: @CharlesIC

### CODE REVIEW

File: config/main_routes.rb [hunk 1]
```diff
@@ -40,13 +47,32 @@ def add_routes(routes_name)
...
+end
+
 constraints IsLoa2 do
   get 'start', to: 'start#index', as: :start
   post 'start', to: 'start#request_post', as: :start
   get 'begin_registration', to: 'start#register', as: :begin_registration
-  get 'choose_a_certified_company', to: 'choose_a_certified_company_loa2#index', as: :choose_a_certified_company
-  post 'choose_a_certified_company', to: 'choose_a_certified_company_loa2#select_idp', as: :choose_a_certified_company_submit
-  get 'choose_a_certified_company/:company', to: 'choose_a_certified_company_loa2#about', as: :choose_a_certified_company_about
+  # HUB-52 Put back when tearing down the no_perf_management AB test
```
Comment (jakubmiarka) [inline config/main_routes.rb]: outdated comment

Comment (CharlesIC) [reply config/main_routes.rb]: Sorry, copied & pasted 😅

Comment (CharlesIC) [reply config/main_routes.rb]: Fixed :)

File: config/main_routes.rb [hunk 2]
```diff
@@ -20,6 +20,13 @@ def add_routes(routes_name)
 get 'may_not_work_if_you_live_overseas', to: 'will_it_work_for_me#may_not_work_if_you_live_overseas', as: :may_not_work_if_you_live_overseas
 get 'will_not_work_without_uk_address', to: 'will_it_work_for_me#will_not_work_without_uk_address', as: :will_not_work_without_uk_address
 
+MOBILE_SEGMENTS = 'mobile_segments'.freeze
+
+mobile_segments_control_piwik = SelectRoute.new(MOBILE_SEGMENTS, 'control', is_start_of_test: true)
```
[the remark this answers is not in the document]
Comment (jakubmiarka) [reply config/main_routes.rb]: for all the SelectRoute instances below

**Pull request merged**